src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -5282,6 +5282,56 @@ contextState switch
                 };
         }
 
+        protected override IParameterSymbol GetCorrespondingParameterCore(SyntaxNode node, CancellationToken cancellationToken)
```

Comment:
>GetCorrespondingParameterCore [](start = 44, length = 29)

I think we usually add a strongly typed version of the API as an extension method, see src\Compilers\CSharp\Portable\CSharpExtensions.cs